Casks/flutter.rb

```diff
@@ -1,6 +1,6 @@
 cask 'flutter' do
-  version "1.5.4-hotfix.2"
-  sha256 "0b4cacbe84fd68cc9cfe213a90ddcdba558e45c19be30c6173c07a318be34014"
+  version "v1.7.8+hotfix.3"
+  sha256 "800db14570c5dd9e4e0d3b403ebd9579cd09226c004239a880470eab438f0872"
```

Comment:
The correct sum is `1bd5d165c79efebfba39e4602d6e07e74b40798630eb7173caedf0f664c53035`. Could you please update that aswell?